### ISSUE TITLE
Angry orange line tooltip

### ISSUE THREAD

**Issue by KevinDCarlson** [ToposInstitute/CatColab#88]
Should be able to float over "spellcheck" lines to get a tooltip telling you what's wrong.

**Comment by epatters** [comment]
Agreed in general about the underline. The orange line in particular is temporary and will be replaced with #58.
